tests/config/rbac_yaml/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 resources:
-- search-e2e-rbac-namespace.yaml
-- search-e2e-rbac-group.yaml
-- search-e2e-rbac-clusterrolebinding.yaml
-- search-e2e-rbac-rolebinding.yaml
+  - search-e2e-rbac-namespace.yaml
+  - search-e2e-rbac-group.yaml
+  - search-e2e-rbac-clusterrolebinding.yaml
+  - search-e2e-rbac-rolebinding.yaml
```

Comment:
Spaces are important in yaml files, so please double check all changes to yaml files.  If needed you can disable yaml from prettier.

Reply:
Correct, this was just adding spaces to the array elements within the YAML file.

Reply:
For now, I believe that this should be okay; however, if there are issues in the future, I will disable the yamls from prettier.